services/vision/mlvision/ml_model.go

```diff
@@ -70,18 +70,33 @@ func registerMLModelVisionService(
 	if err != nil {
 		return nil, err
 	}
+
 	classifierFunc, err := attemptToBuildClassifier(mlm)
 	if err != nil {
-		logger.Infow("error turning turn ml model into a classifier", "model", params.ModelName, "error", err)
+		logger.Infow("error turning ml model into a classifier", "model", params.ModelName, "error", err)
 	} else {
-		logger.Infow("model fulfills a vision service classifier", "model", params.ModelName)
+		err := checkIfClassifierWorks(ctx, classifierFunc)
```

Comment:
Definitely appreciate the edits to unpack().  Not gonna lie, I am a little confused about this setup. Like, when do we expect the check() test to work but not attemptToBuild()?